node-runner-cli/utils/utils.py

```diff
@@ -214,9 +214,8 @@ def merge(source, destination):
     def parse_trustednode(trustednode):
         import re
 
-        if not bool(
-            re.match(r"radix://(.*)@\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$", trustednode)
-        ):
+        regex = r"radix://(.*)@((\d{1,3}\.){3}\d{1,3}|([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,})$"
```

Comment:
Do we not have default hardcoded value? That might need to be changed as well

Reply:
@shambupujar can you elaborate? The hardcoded regex or a hardcoded default trusted node?

Reply:
Refers to this https://github.com/radixdlt/babylon-nodecli/pull/125/files#diff-4613f3d39a2065bf343d9e146fe5ebfdeff0daea4c1b585747c030e59c91d1c4R271